index.html

```diff
@@ -1,13 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <!-- Don't forget to change the title and connect the CSS file -->
-  <title>Project Library</title>
-</head>
-<body>
-  <h1>Project Library</h1>
-  <!-- Don't forget to connect the JavaScript file -->
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@400;700&family=Jost:ital,wght@0,100..900;1,100..900&family=Montserrat&family=Schoolbell&display=swap"
+      rel="stylesheet"
+    />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>That cool Library</title>
+  </head>
+
+  <body>
+    <h1>That cool Library</h1>
+    <section id="filterBar" class="filter-bar">
+      <div id="filtersOnGenre" class="filter-genre">
+        <h2>Filter by Genre</h2>
+        <button id="allBooks" class="filter-genre" onclick="showLibrary(books)">
+          All
+        </button>
+        <button id="filterFiction" onclick="fictionSelect(books)">
+          Fiction
```

Comment:
I think if we've already used `addEventListener` on JS, we don't need to add `onclick` on html again?